snippets/reducedFilter.md

```diff
@@ -0,0 +1,38 @@
+### reducedFilter
+
+Filter an array of objects based on condition and return array with reduced objects.
+
+#### Input
+
+* Data: the data to be filtered (array of objects) 
+* Condition: will be used for filtering (string) 
+* outputProps: an array of properties that will be used to contruct new array of objects 
+
+#### Output
+
+* Filtered array with new objects. Properties of new objects are a subset of
+properties of original objects 
+
+#### Info
+
+Used ES6 reduce 
+Dummy data for testing 
+Generated with http://www.mockaroo.com/ 
+
```

Comment:
Explain more thoroughly the techniques and methods used.

Reply:
Hello @Chalarangelo. I added more details in the update files. Could you please review it? Thanks.